### ISSUE TITLE
Error when sending funds on Matic (prefix MATIC)

### ISSUE THREAD

**Issue by pscott** [MetaMask/metamask-extension#11086]
**Describe the bug**

Filling an address on Matic network when trying to send funds errors:

```
Message: This method only supports 0x-prefixed hex strings but input was: MATIC

Code: Error

Stack:
Error: This method only supports 0x-prefixed hex strings but input was: MATIC
  at Object.r.assertIsHexString (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui.js:72:210305)
  at Object.r.toChecksumAddress (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui.js:72:201511)
  at _ (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui.js:152:653608)
  at h.render (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui.js:152:478626)
  at Ya (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:70734)
  at Ka (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:70529)
  at jl (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:105704)
  at bu (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:90053)
  at pu (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:89978)
  at iu (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:87326)
  at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:45768
  at r.unstable_runWithPriority (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:172:3472)
  at Vo (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:45477)
  at Yo (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:45713)
  at Ko (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:45648)
  at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:114643
  at ue (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:114649)
  at fe (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:5411)
  at Rr (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:29682)
  at Cr (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:30610)
  at Tr (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:29898)
  at r.unstable_runWithPriority (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:172:3472)
  at Vo (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:45477)
  at le (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:114479)
  at Er (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-libs.js:75:29321)
  at HTMLDocument.c (chrome-extension://
  ```

This is whether I use `transfer between account`, paste, or put the address by hand.

**Steps to reproduce (REQUIRED)**
Metamask on Chrome v9.5.1

![Screenshot from 2021-05-13 19-48-16](https://user-images.githubusercontent.com/30843220/118165105-366f0580-b424-11eb-889a-419a7068561f.png)

**Browser details (please complete the following information):**
 - Linux
 - Ledger Nano X firmware 1.2.4-5
 - Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)
 - MetaMask Version: 9.5.1

**Comment by akhanubis** [comment]
Having the same issue, can't send MATIC on Polygon at all

**Comment by gallantpui** [comment]
MetaMask Version on chrome n firefox is on v9.5.2 though. Are you seeing the error with on this ver ?

**Comment by brad-decker** [comment]
Fixed in 9.5.2, thanks for reporting!
